lib/ui/src/containers/preview.js

```diff
@@ -20,6 +20,10 @@ const mapper = ({ api, state: { layout, location, path, storyId, viewMode, selec
     : {
         api,
         getElements: api.getElements,
+        getQueryParams: () =>
+          Object.entries(api.getUrlState().queryParams).reduce((acc, [k, v]) => {
+            return `${acc}&${k}=${v}`;
+          }, ''),
```

Comment:
I was wondering for a while until I see the leading `&` was added here. Not so intuitive but I think that is okay since it become extendable by default

Reply:
I think it would make more sense to pass the raw API (`getUrlState()`) down the the preview and have it do the putting it together into a string (probably using `import { URL } from 'url'` too!)

Reply:
ok